includes/html/pages/pseudowires.inc.php

```diff
@@ -39,7 +39,8 @@
 foreach (dbFetchRows('SELECT * FROM pseudowires AS P, ports AS I, devices AS D WHERE P.port_id = I.port_id AND I.device_id = D.device_id ORDER BY D.hostname,I.ifDescr') as $pw_a) {
     $pw_a = cleanPort($pw_a);
     $i = 0;
-    while ($i < count($linkdone)) {
+//    while ($i < count($linkdone)) { //php8 needed this line changed -st
```

Comment:
please dont leave commented code

Reply:
What is with this old code using a while when a foreach loop was clearly more suited?

Reply:
I don't know.  All of a sudden the error popped up.  I constantly got an error about a countable array.  From what I found, this seemed to be the answer.  Sorry about the inline comments.  This is my first time trying to help.

Reply:
@scottbob09 my comment was not directed towards you, but original authors.  Thanks for sending a fix.

Reply:
I was just a little frustrated that this needed to be fixed in the first place.  If it would have used foreach, it would not have needed a fix.